tools.go

```diff
@@ -0,0 +1,29 @@
+//go:build tools
+// +build tools
+
+/*
+Copyright 2021 The Kubernetes-CSI-Addons Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// These imports are used to maintain a copy of the used tools in the vendor/
+// directory. Versions of the tools can automatically be updated, just like
+// other dependencies. It also makes things easier to install the tools in
+// environments that have slow networking or are completely disconnected.
+
+package main
+
+import (
+	_ "sigs.k8s.io/kustomize/kustomize/v4"
```

Comment:
if kustomize is updated to v5 we need changes here also. will dependent bot takes care of it?

Reply:
no, I do not think so. But The way it is currently done (version in the `Makefile`) will not even see minor updates.